src/lib/select/select.spec.ts

```diff
@@ -29,6 +29,7 @@ import {
   FloatPlaceholderType,
   MD_PLACEHOLDER_GLOBAL_OPTIONS
 } from '../core/placeholder/placeholder-options';
+import 'rxjs/add/operator/map';
```

Comment:
Given that we are trying to move away from this syntax, can you refactor to use the explicit symbol?